run_page/codoon_sync.py

```diff
@@ -149,7 +153,24 @@ def tcx_output(fit_array, run_data):
         },
     )
     # xml tree
-    tree = ET.ElementTree(training_center_database)
+    ET.ElementTree(training_center_database)
+    # Author
+    author = ET.Element(
+        "Author",
+        {
+            "xsi:type": "Application_t"
+        }
+    )
+    training_center_database.append(author)
+    author_name = ET.Element("Name")
+    author_name.text = "Connect Api"
+    author.append(author_name)
+    author_lang = ET.Element("LangID")
+    author_lang.text = "en"
+    author.append(author_lang)
+    author_part = ET.Element("PartNumber")
+    author_part.text = "006-D2449-00"
```

Comment:
Extract this string to a variable and leave a comment specifying which Garmin device it is for.